client-course-schedulizer/src/index.tsx

```diff
@@ -15,3 +16,4 @@ ReactDOM.render(
 // unregister() to register() below. Note this comes with some pitfalls.
 // Learn more about service workers: https://bit.ly/CRA-PWA
 serviceWorker.unregister();
+read.readCSV();
```

Comment:
This is a great way to test the readCSV, but generally, we will not edit `index.tsx`. Somewhere in the `<App />` component, there will be something that calls `read.readCSV()` once a file is uploaded. We can remove this when we add that functionality.

Reply:
To be addressed in #21